cs/build/internal/Common.Internal.targets

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="'$(TargetFrameworkProfile)' != 'Profile78'">
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' != 'Profile78'">
```

Comment:
When does `TargetFrameworkProfile` get set anymore? Can we delete the `Profile78` stuff?

Reply:
It breaks the Compiler.csproj if I remove the check for TargetFrameworkProfile